Source/Scene/PointCloud3DTileContent.js

```diff
@@ -991,20 +1003,26 @@ define([
         }
 
         if (hasColorStyle) {
-            vs += '    color = getColorFromStyle(position, position_absolute, color, normal); \n';
+            vs += '    color = getColorFromStyle(position, positionWC, color, normal); \n';
         }
 
         if (hasShowStyle) {
-            vs += '    float show = float(getShowFromStyle(position, position_absolute, color, normal)); \n';
+            vs += '    float show = float(getShowFromStyle(position, positionWC, color, normal)); \n';
         }
 
         if (hasPointSizeStyle) {
-            vs += '    gl_PointSize = getPointSizeFromStyle(position, position_absolute, color, normal); \n';
+            vs += '    gl_PointSize = getPointSizeFromStyle(position, positionWC, color, normal); \n';
         } else {
             vs += '    gl_PointSize = u_pointSize; \n';
         }
 
-        vs += '    color = color * u_highlightColor; \n';
+        if (pointAttenuation) {
+            vs += '    vec4 positionEC = czm_view * vec4(positionWC, 1.0); \n' +
```

Comment:
Yeah I could see it going either way. The one area that would be more complicated with defines is the attribute declaration section, but otherwise it falls pretty neatly into defines.